src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -48,23 +57,74 @@
 import io.sphere.sdk.types.commands.TypeDeleteCommand;
 import io.sphere.sdk.types.queries.TypeQuery;
 import java.time.Clock;
+import java.time.ZonedDateTime;
 import javax.annotation.Nonnull;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import uk.org.lidalia.slf4jtest.TestLogger;
 import uk.org.lidalia.slf4jtest.TestLoggerFactory;
 
 class CliRunnerIT {
+
   private static final TestLogger syncerTestLogger = TestLoggerFactory.getTestLogger(Syncer.class);
   private static final TestLogger cliRunnerTestLogger =
       TestLoggerFactory.getTestLogger(CliRunner.class);
+  private static final String RESOURCE_KEY = "foo";
+
 
   @BeforeEach
   void setup() {
     syncerTestLogger.clearAll();
     cliRunnerTestLogger.clearAll();
     cleanUpProjects(createClient(CTP_SOURCE_CLIENT_CONFIG), createClient(CTP_TARGET_CLIENT_CONFIG));
+    setupSourceProjectData(createClient(CTP_SOURCE_CLIENT_CONFIG));
+  }
+
+  static void setupSourceProjectData(@Nonnull final SphereClient sourceProjectClient) {
+    final ProductTypeDraft productTypeDraft =
+        ProductTypeDraftBuilder.of(RESOURCE_KEY, "bar", "desc", emptyList()).build();
```

Comment:
in general this sample values like "foo", "bar" is not helpful (imo)

Reply:
f028a4f